localplanning.services/src/components/LpaMasthead.astro

```diff
@@ -0,0 +1,39 @@
+---
+import Container from "@components/Container.astro";
+import { Image } from "astro:assets";
+
+interface Props {
+  title: string;
+  lpaName: string;
+  backgroundColor?: string;
+  primaryColor?: string;
```

Comment:
It's safe to consider `primaryColor` as required props as we set defaults when generating a new team in PlanX - each team will always have a value.